src/style.css

```diff
@@ -0,0 +1,161 @@
+/* #region reset */
```

Comment:
Just kindly recommendation. It's better to name it `section reset`, not 'region' if you want to markup different style sections.